packages/lab/src/list/List.tsx

```diff
@@ -82,7 +82,7 @@ export const List = forwardRef(function List<
     width,
     ...htmlAttributes
   }: ListProps<Item, Selection>,
-  forwardedRef?: ForwardedRef<ScrollingAPI<Item>>
+  forwardedRef?: ForwardedRef<HTMLDivElement>
```

Comment:
This makes the type of the forwardRef correct, as I'm using the forwardedRef for the imperative API. One thing to be aware of - there was some discussion about whether List should really do this, or set the forwarded ref on the html container, to make it mor similar to other components. We would then have to return the scrolling API via a custom ref.

David wanted the latter behaviour, because it would work better for App Builder. Can't remember what the consensus was, but as toy see, I didn't actually change it.